yazi-plugin/preset/ya.lua

```diff
@@ -50,3 +50,14 @@ function ya.readable_path(path)
 		return path
 	end
 end
+
+function ya.owner(cha)
+	if not cha.uid or not cha.gid then
+		return nil
+	end
+
+	user = ya.user_name(cha.uid) or cha.uid
+	group = ya.group_name(cha.gid) or cha.gid
```

Comment:
I wanted to write this helper function to handle possibly missing `uid` and `gid` values. It incidentally fixes a bug in Yazi where the linemode would fail if `uid` was available, but the user name was not.